app/src/main/java/com/example/myapplication/FirstFragment.java

```diff
@@ -94,19 +91,23 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
 
             @Override
             public void afterTextChanged(Editable s) {
+                if (s.length() > 5){
```

Comment:
there currently aren't any five digit teams so this can be changed to only allow four digit teams, and then next year you can update it to allow five digit teams

Reply:
ok